### PR TITLE
Next-Gen apps docs & release notes (GA: week ending 6/23)

### PR DESCRIPTION
**Pull request by kevin-sumo**
Docs and release notes for our Next-Gen Apps release.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-06-15-next-gen-apps.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+In response to many requests for better App management, we are happy to announce the release of our new Next-Gen Apps located within a new section of the Applications Catalog. Sumo Logic's next generation of Apps introduce features not previously available within Classic Apps and simplifies the management and maintenance of Apps. 
+
+**Key Features**
+
+* In order to know which Apps have been installed within your account these new apps will provide a badge noting their installed status. 
+* When an update to an installed App becomes available these new apps will provide a badge noting their is an update available. An upgrade can then be applied with a simple click within the App Catalog.
```
Comment (jpipkin1) [inline blog-service/2023-06-15-next-gen-apps.md:18]:
```suggestion
* When an update to an installed App becomes available, these new apps will provide a badge noting there is an update available. An upgrade can then be applied with a simple click within the App Catalog.  
```

File: blog-service/2023-06-15-next-gen-apps.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+In response to many requests for better App management, we are happy to announce the release of our new Next-Gen Apps located within a new section of the Applications Catalog. Sumo Logic's next generation of Apps introduce features not previously available within Classic Apps and simplifies the management and maintenance of Apps. 
+
+**Key Features**
+
+* In order to know which Apps have been installed within your account these new apps will provide a badge noting their installed status. 
+* When an update to an installed App becomes available these new apps will provide a badge noting their is an update available. An upgrade can then be applied with a simple click within the App Catalog.  
+* Installed Apps can be uninstalled and removed through the App Catalog, making it easy to clean up Apps you no longer use.  
+* Additional filter options making it easy to find your installed or upgradeable Apps.
```
Comment (jpipkin1) [inline blog-service/2023-06-15-next-gen-apps.md:20]:
```suggestion
* Additional filter options make it easy to find your installed or upgradeable Apps. 
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@kevin-sumo, I made a few more suggestions. 

Also, I think you should make "apps" lowercase when the word stands by itself. Capitalize the word only when it is part of an actual proper noun, like Classic Apps, Next-Gen Apps, and App Catalog.

**Comment by jpipkin1** [comment]
Thanks for changing "Apps" to "apps" where appropriate, @kevin-sumo. Looks great.

**Review by jpipkin1 — APPROVED** [review]

File: blog-service/2023-06-15-next-gen-apps.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* Role-based access controls, which ensure that only authorized individuals within your organization can install, uninstall, or update apps. To perform these actions you must be a member of the default Administrator role. Sumo Logic will be providing a new “Manage Apps” role capability soon, which can be used to grant these permissions to additonal roles.
+
+While the initial set of new apps is designed to help you get started using the Sumo Logic distribution of the Open Telemetry agent, we haven't forgotten about those who may continue to send data through the Sumo Logic Installed Collector and Hosted Sources. We will be converting all the existing Classic Apps to the next gen framework in the coming months and will continue to provide support for these collection methods. In the meantime, our Classic Apps will remain available to you within the Application Catalog.
+
+Over the coming year Sumo Logic will also be making additional improvements within the Applications Catalog to assist with better search, filtering, upgrade notifications, and management. Please lookout for these future updates.
```
Comment (kimsauce) [inline blog-service/2023-06-15-next-gen-apps.md:31]:
```suggestion
Over the coming year, Sumo Logic will be making additional improvements within the Apps Catalog to assist with better search, filtering, upgrade notifications, and management. Stay tuned for these future updates. 
```

File: docs/get-started/apps-integrations.md [hunk 4]
```diff
@@ -74,7 +98,8 @@ Each Sumo Logic App has unique requirements, so make sure to follow the specific
    * In the **Install Application** dialog, indicate the name of the environment in the Source Host, such as `_sourceHost=perf`.
    * Rename the title of the App to denote the environment it is configured for, for example, **Windows-Perf**.
 
-### Troubleshooting
+
```
Comment (kimsauce) [inline docs/get-started/apps-integrations.md:101]:
```suggestion
```

**Pull request merged**